src/Orleans/Streams/PersistentStreams/IStreamQueueBalancer.cs

```diff
@@ -37,7 +42,7 @@ internal interface IStreamQueueBalancer
     /// It should be implemented by components interested in stream queue load balancing.
     /// When change notification is received, listener should request updated list of queues from the queue balancer.
     /// </summary>
-    internal interface IStreamQueueBalanceListener : IAddressable
+    public interface IStreamQueueBalanceListener : IAddressable
```

Comment:
it was IAddressable before. But you are right, this limits listener to be only IAddressable.

I wonder is there historical reason for it @jason-bragg ?